src/platform/site-wide/mega-menu/components/MegaMenu.jsx

```diff
@@ -18,6 +18,9 @@ export default class MegaMenu extends React.Component {
 
     this.mobileMediaQuery.addListener(this.resetDefaultState);
     document.body.addEventListener('click', this.handleDocumentClick, false);
+
+    this.signInVis =
+      window.location.pathname === '/sign-in/' ? 'hidden' : 'visibile';
```

Comment:
No, remove it. Again lets use CSS in our SignInApp to update styles.

Reply:
updated